### PR TITLE
Fix switcher and filter button styling issues in MyList page

### PR DESCRIPTION
**Pull request by danyiliev**
- My/List Archive switcher styling
- Filter button styling

### CODE REVIEW

File: PocketKit/Sources/PocketKit/Home/TopicChipCell.swift [hunk 1]
```diff
@@ -71,6 +88,7 @@ extension TopicChipCell {
 
     static func width(chip: TopicChipPresenter) -> CGFloat {
         let size = chip.attributedTitle?.sizeFitting() ?? .zero
-        return size.width.rounded(.up) + 24
+        let imageSize = chip.iconImage?.size.width ?? .zero
```
Comment (jacobsimeon) [inline PocketKit/Sources/PocketKit/Home/TopicChipCell.swift:91]: Use `24` points of padding if imageSize is `0`

File: PocketKit/Sources/PocketKit/MyList/MyListTitleView.swift [hunk 2]
```diff
@@ -58,10 +57,11 @@ class MyListTitleView: UIView {
     
     override func layoutSubviews() {
         super.layoutSubviews()
-        
+
+        selection.backgroundColor = UIColor(.ui.teal6)
         selection.frame = lastSelectedButton?.frame ?? .zero
         selection.layer.cornerRadius = selection.frame.height / 2
-        selection.layer.borderColor = UIColor(.ui.grey1).cgColor
+        selection.layer.borderColor = UIColor.clear.cgColor
```
Comment (jacobsimeon) [inline PocketKit/Sources/PocketKit/MyList/MyListTitleView.swift:64]: Let's go ahead and remove the border here

**Review by cyndichin — COMMENTED** [review]
Looks good so far! Comparing to the Figma file, I don't see a border - should we get rid it? Also, I feel that `My List` isn't the same font style.

simulator_screenshot_E8915A2B-FAB5-47C0-9A9E-A56DC9668D12

**Review by jacobsimeon — APPROVED** [review]
Approved, pending the two comments I've added inline

File: PocketKit/Sources/PocketKit/Home/TopicChipCell.swift (hunk 1 above)
Comment (danyiliev) [reply PocketKit/Sources/PocketKit/Home/TopicChipCell.swift:91]: Fixed as requested.

**Comment by danyiliev** [comment]
> Looks good so far! Comparing to the Figma file, I don't see a border - should we get rid it? Also, I feel that `My List` / `Archive` isn't the same font style.
> 
> simulator_screenshot_E8915A2B-FAB5-47C0-9A9E-A56DC9668D12

Thanks you for your comment. @cyndichin the borders were cleared. The different font issue was already discussed with Jacob. It is escalated to our designer.

**Review by dskuza — APPROVED** [review]
Looks good to me!

**Comment by audie1** [comment]
No issues found on phone or tablet, icon looks good, QA Pass

**Pull request merged**